### ISSUE TITLE
Extension node not found...

### ISSUE THREAD

**Issue by harry-cpp** [sevoku/MonoDevelop.Xwt#2]
Error on startup of MonoDevelop:

```
MonoDevelop.Xwt.1.1
    Full Path: MonoDevelop.Xwt,1.1.0
    Error: Extension node not found or not extensible: /MonoDevelop/Ide/ProjectTemplateCategories/crossplat/app
```

I suspect that the extension does not work because of the newest MonoDevelop update.

**Comment by thiagojedi** [comment]
It works in Xamarin Studio, but not in MonoDevelop. Maybe there's something to do with XS crossplatform templates?

**Comment by harry-cpp** [comment]
Yup, does not exist in MonoDevelop: https://github.com/mono/monodevelop/blob/2eac19bc2d992053001b077c6576ae94666a249a/main/src/core/MonoDevelop.Ide/ExtensionModel/Templates.addin.xml#L96

**Comment by sevoku** [comment]
Thanks for reporting and sorry that I missed the notification! Should be fixed with 0f69bcff371750e2a65ca7ee2442b1a5d923bfc2
